<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogWPAEnterprise</class>
 <widget class="QDialog" name="dialogWPAEnterprise">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WPA-Enterprise Configuration</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/tray_wifi85.png</normaloff>:/tray_wifi85.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>WPA Enterprise Configuration</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="textLabel1">
        <property name="text">
         <string>EAP authentication method</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QRadioButton" name="radioEAPTLS">
          <property name="text">
           <string>EAP-TLS</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioEAPTTLS">
          <property name="text">
           <string>EAP-TTLS</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioEAPPEAP">
          <property name="text">
           <string>EAP-PEAP</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="textLabel2">
        <property name="text">
         <string>EAP Identity:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEAPIdentity"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="textCACert">
        <property name="text">
         <string>CA Certificate:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineCACert">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="pushSelectCACert">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="libpcbsd-ui.qrc">
          <normaloff>:/folder_open.png</normaloff>:/folder_open.png</iconset>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="textClientCert">
        <property name="text">
         <string>Client Certificate:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineClientCert">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="pushSelectClientCert">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="libpcbsd-ui.qrc">
          <normaloff>:/folder_open.png</normaloff>:/folder_open.png</iconset>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="textPrivateKeyFile">
        <property name="text">
         <string>Private Key File:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="linePrivateKeyFile">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QPushButton" name="pushSelectPrivateKeyFile">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="libpcbsd-ui.qrc">
          <normaloff>:/folder_open.png</normaloff>:/folder_open.png</iconset>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="textPrivateKey">
        <property name="text">
         <string>Password:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="2">
       <widget class="QLineEdit" name="linePrivateKeyPassword">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="textPrivateKey2">
        <property name="text">
         <string>Repeat Password:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="1" colspan="2">
       <widget class="QLineEdit" name="linePrivateKeyPassword2">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="3">
       <widget class="QCheckBox" name="checkShowKey">
        <property name="text">
         <string>Show Key</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboKeyMgmt">
        <item>
         <property name="text">
          <string notr="true">WPA-EAP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">IEE8021X</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelKeyMgmt">
        <property name="text">
         <string>Key Management</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="spacer8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>261</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushClose">
       <property name="text">
        <string>&amp;Close</string>
       </property>
       <property name="shortcut">
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <includes>
  <include location="local">qmessagebox.h</include>
 </includes>
 <resources>
  <include location="libpcbsd-ui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>radioEAPTLS</sender>
   <signal>clicked()</signal>
   <receiver>dialogWPAEnterprise</receiver>
   <slot>slotTypeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioEAPTTLS</sender>
   <signal>clicked()</signal>
   <receiver>dialogWPAEnterprise</receiver>
   <slot>slotTypeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioEAPPEAP</sender>
   <signal>clicked()</signal>
   <receiver>dialogWPAEnterprise</receiver>
   <slot>slotTypeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushClose</sender>
   <signal>clicked()</signal>
   <receiver>dialogWPAEnterprise</receiver>
   <slot>slotClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushSelectCACert</sender>
   <signal>clicked()</signal>
   <receiver>dialogWPAEnterprise</receiver>
   <slot>slotSelectCACert()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushSelectClientCert</sender>
   <signal>clicked()</signal>
   <receiver>dialogWPAEnterprise</receiver>
   <slot>slotSelectClientCert()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushSelectPrivateKeyFile</sender>
   <signal>clicked()</signal>
   <receiver>dialogWPAEnterprise</receiver>
   <slot>slotSelectPrivateKeyFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkShowKey</sender>
   <signal>clicked()</signal>
   <receiver>dialogWPAEnterprise</receiver>
   <slot>slotShowKey()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
